mlir/utils/jenkins/Jenkinsfile

```diff
@@ -972,10 +972,10 @@ pipeline {
                                 timeout(time: 60, activity: true, unit: 'MINUTES') {
                                     withEnv(['MIGRAPHX_ENABLE_MLIR=1']) {
                                         // Verify MLIR unit tests
-                                        sh 'make -j$(nproc) test_gpu_mlir'
+                                        sh 'make -j$(nproc) driver test_gpu_mlir'
```

Comment:
`test_gpu_mlir` wont build the driver. You need to build it explicitly.

Reply:
@pfultz2 https://github.com/ROCmSoftwarePlatform/rocMLIR/blob/3116b350d684f3aefbce98b154f737f8610095c8/mlir/utils/jenkins/Jenkinsfile#L52 

This would build `driver`. I don't see need to add it  explictly.